modules/coreos-layering-configuring.adoc

```diff
@@ -147,8 +147,8 @@ $ oc get mcp
 [source,terminal]
 ----
 NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
-master   rendered-master-6faecdfa1b25c114a58cf178fbaa45e2   True      False      False      3              3                   3                     0                      39m
-worker   rendered-worker-6b000dbc31aaee63c6a2d56d04cd4c1b   False     True       False      3              0                   0                     0                      39m <1>
+master   rendered-master-15961f1da260f7be141006404d17d39b   True      False      False      3              3                   3                     0                      39m
+worker   rendered-worker-5de4837625b1cbc237de6b22bc0bc873   False     True       False      3              0                   0                     0                      39m <1>
```

Comment:
The worker pool will not show the new configuration until all the workers are configured

Since in this command there are worker nodes pending for configuration, the "oc get mcp" will not show the latest rendered machineconfig yet. It shows the previous one `rendered-worker-5aff604cb1381a4fe07feaf1595a797e`

One the configuration is fully applied, than the `oc get mcp` will show the latest config

`worker   rendered-worker-5de4837625b1cbc237de6b22bc0bc873   False     True       False      3              3                   3                     0                      39m <1>`

Reply:
thank you Sergio, i've updated this